### DISCUSSION TITLE
Making it all work - some hints

### DISCUSSION THREAD

**Discussion by sahafeez** [kuestess/homebridge-platform-insteonlocal#238] [General]
@kuestess  removed the comment here as I am a bit confused between controllers / groupMember now.

Let us say I have 2 switches. They are Insteon linked (each is a responder/controller for each other).

```
                {
                    "name": "Dining.Kitchen",
                    "deviceID": "41.2B.95",
                    "dimmable": "yes",
                    "deviceType": "dimmer",
                    "controllers": [
                        "4118DF"
                    ],
                    "groupMembers": "412B95,4118DF"
                },
                {
                    "name": "Dining.Patio",
                    "deviceID": "41.18.DF",
                    "dimmable": "yes",
                    "deviceType": "dimmer",
                    "controllers": [
                        "412B95"
                    ],
                    "groupMembers": "412B95,41118DF"
                },
```

1. In reading the documentation it seems that I would need both the controllers and groupMemebers configuration lines?

2. Also I assume that the PLM/HUB is always considered a controller by default?

3. For a scene, I have added the groupMembers to the scene configuration with all the IDs of it's members. Is that correct?

**Comment by 64Spaces** [comment]
I have the same general question (see my other post). GroupMember seems to update the status of the devices in the scene. But I’m not sure what Controllers does. The scene devices seem to be updated with or without the Controller setting.

**Comment by JimCSaratoga** [comment]
I have this same configuration on my Kitchen Table Lights.  I've been able to get it to work with just the controllers (leaving the groupMembers out because there isn't a groupID referenced).  In the past with the Insteon app there was the ability to set these up in a "3-way" config explicitly building a scene with it (and getting the associated 'scene ID' with it.  If I've understood your initial post @sahafeez I think you've manually paired in both directions (bi-directional controller/responder set button pairing) the two dimmers which is the same as I have done and it works.  Home can control them and the switches can control the home status indication (I did group the two dimmers in the home app).  Hope this helps in some manner.
